src/CBT.NoTarget/build/NoTarget.targets

```diff
@@ -7,6 +7,13 @@
 
   <Import Project="$(CustomBeforeNoTargets)" Condition="'$(CustomBeforeNoTargets)' != '' and Exists('$(CustomBeforeNoTargets)')" />
 
+  <ItemDefinitionGroup Condition="'$(NoTargetsCopyReferenceOutput)' == 'true'" >
```

Comment:
I think the name of the property should be `NoTargetsReferenceOutputAssembly` and check here if the user set it to `false`.  What do you think?

Reply:
Agreed, that's better. Updating.